### PR TITLE
Remove dist folder

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Stop checking in the dist folder as part of aspnet/Coherence-Signed#953.

Also some minor cleanup and adding a VSTS pipeline yml file.

### CODE REVIEW

**Comment by Eilon** [comment]
Hold on. Trying to think about this. Doesn't one of the packaging types require a Git repo with the product in it? Or was that Bower? What does NPM require in the Git repo? Anything? My brain is super fuzzy on this.

**Comment by ryanbrandenburg** [comment]
[This doc](https://github.com/aspnet/jquery-validation-unobtrusive/wiki/Release-checklist) reads to me that the sources of truth are:

|Source | Truth  |
|--------|--------|
|NPM    | tgz      |
|NuGet  | nupkg |
|cdnjs    | NPM   |

**Comment by ryanbrandenburg** [comment]
Some quick googling didn't turn up any publishing schemes that use your git repository. @SteveSandersonMS and/or @kichalla, do you guys know anything about that?

**Comment by anurse** [comment]
Bower used to use Git as the "storage" for packages, but it's been long deprecated. I don't see any problem with removing the `dist` folder.

**Comment by Eilon** [comment]
OK that's good then. It was probably Bower.

**Comment by anurse** [comment]
Bye bye bowser

Bowser throw

**Comment by ryanbrandenburg** [comment]
... he never lets go.

**Comment by anurse** [comment]
No, he definitely does. Just keep watching.

File: .vsts-pipelines/builds/ci-public.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+resources:
+  repositories:
+  - repository: buildtools
+    type: github
+    endpoint: DotNet-Bot GitHub Connection
+    name: aspnet/BuildTools
+    ref: refs/heads/release/2.2
+
+phases:
+  - template: .vsts-pipelines/templates/phases/default-build.yml@buildtools
```
Comment (SteveSandersonMS) [inline .vsts-pipelines/builds/ci-public.yml]: I don't know where this comes from. Is it part of our internal SDK, or built into VSTS?

Comment (anurse) [reply .vsts-pipelines/builds/ci-public.yml]: It's part of the internal SDK, the `@buildtools` references the `repositories` section above where we link in https://github.com/aspnet/BuildTools . All our other repos use this same template.

Comment (ryanbrandenburg) [reply .vsts-pipelines/builds/ci-public.yml]: We're not actually using this part yet anyway due to a problem in VSTS.

**Pull request merged**